.circleci/config.yml

```diff
@@ -272,7 +272,7 @@ jobs:
             free
             trap "go-junit-report </go/out/tests/go-test-report.out > /go/out/tests/go-test-report.xml" EXIT
             make localTestEnv
-            make pilot-test mixer-test broker-test security-test T=-v | tee -a /go/out/tests/go-test-report.out
+            make pilot-test mixer-test broker-test security-test common-test T="-v -p 1" | tee -a /go/out/tests/go-test-report.out
```

Comment:
This will make it pointless to have a large instance and run only one test at a time

Reply:
The proper way is to use the CI 'context' env variables - we can leave the default matching a large image.

The problem is that even on a large image I've seen failures - I don't think the default (guessed by go) is good enough. But we can adjust.